src/test/Assets/TestProjects/PortableTestApp/PortableTestApp.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>PortableTestApp</AssemblyName>
+    <PackageId>PortableTestApp</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PackageTargetFallback>$(PackageTargetFallback);dotnet5.4;portable-net451+win8</PackageTargetFallback>
```

Comment:
Do we know why this `PackageTargetFallback` is necessary?  I don't see it on others